src/runtime_src/core/common/time.h

```diff
@@ -19,6 +19,7 @@
 
 #include "core/common/config.h"
 #include <string>
+#include <ctime>
```

Comment:
Please don't include unnecessary header files.  You should include exactly what you need in order to compile the header file stand-alone and <ctime> is not needed.  Reverted in #4259